roles/rust-foundation-project-director.md

```diff
@@ -1,36 +1,92 @@
 # Role Description: Rust Foundation Project Director
 
-Please refer to [Rust Foundation Director Roles & Responsibilities][director-role] for components of the role description that apply to all Foundation Board Directors.
+A Project Director is one of five Rust Project members who serve on the Board of Directors[^directors] of the Rust Foundation to represent the interests of the Rust Project[^establishment]. Project Directors are elected per the [Project Director Election Process].
 
-Project Directors are members of the Rust Foundation Board of Directors.
-The Board of Directors is the governing body of the Rust Foundation, which is incorporated as a 501(c)(6) organization in the State of Delaware, United States of America.
-As such, the Project Director role carries with it legal responsibilities to act on behalf of and in support of the Foundation, as well as some legal liability for the Foundation's actions[^liability].
+The Board of Directors is the governing body of the Rust Foundation, which is incorporated as a 501(c)(6) organization in the State of Delaware, United States of America. As such, the Project Director role carries with it legal responsibilities to act on behalf of and in support of the Foundation, as well as some legal liability for the Foundation's actions[^liability].
 
-The Board of Directors sets the direction of the Rust Foundation.
-Among the mechanisms the Board has to do this is hiring and overseeing the Executive Director, who is responsible for the day-to-day operations of the Foundation.
+The Board of Directors sets the direction of the Rust Foundation. Among the mechanisms the Board has to do this is hiring and overseeing the Executive Director, who is responsible for the day-to-day operations of the Foundation.
 
+[^directors]: Please refer to [Rust Foundation Director Roles & Responsibilities] for components of the role description that apply to all Foundation Board Directors.
 [^liability]: While this sounds daunting, the Foundation has insurance to protect the Directors from legal action brought against the faction. As long as Directors carry out their board duties in good faith, to the best of their abilities, and with the best interests of the Foundation in mind, they are protected by this insurance.
+[^establishment]: The Project Directors specifically are established by the [Section 4.3(d)] of the Foundation's By-laws to represent the "Individual Members" of the Foundation. Individual members are defined as those in "the governing body of the Rust project" ([Section 2.2]), currently the Leadership Council.
+
+    By analogy, Project Directors are to the Leadership Council what Google's Platinum Member Director is to Google, for example. This isn't exactly right, as the Rust Project is distinct as a constituent of the Foundation, as the Foundation exists explicitly to serve the Rust Project, but does not exist to serve its member organisations (outwith stipulations of membership)
```

Comment:
```suggestion
    By analogy, Project Directors are to the Leadership Council what Google's Platinum Member Director is to Google, in the sense that they represent the Council's (and by extension the Project's) interests in the Rust Foundation. Although it should be noted the Rust Project is distinct from member organizations of the Rust Foundation, as the Foundation exists explicitly to serve the Rust Project, but does not exist to serve its member organisations (outwith stipulations of membership)
```
Maybe that's a bit clearer? It sounded weird to have "this isn't exactly right" in the sentence.